### ISSUE TITLE
[Problem] Inverse Shortening of Center Lines

### ISSUE THREAD

**Issue by frank-trampe** [FreeCAD/FreeCAD#12691]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Attempting to shorten a cosmetic line in TechDraw makes the line longer. This problem is present in 0.21.2 also.

### Full version info

```shell
OS: Ubuntu 20.04.6 LTS (ubuntu:GNOME/ubuntu)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36082 (Git)
Build type: Release
Branch: main
Hash: 7a025e4e60a607bdbdd1599627ce13bb8eeda071
Python 3.10.13, Qt 5.15.8, Coin 4.0.2, Vtk 9.2.6, OCC 7.6.3
Locale: English/United States (en_US)
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
Can confirm on Windows too

**Comment by frank-trampe** [comment]
Huh. That's odd. Shortening those works.

**Comment by WandererFan** [comment]
Is this using one of the extension functions (red icons)?

**Comment by frank-trampe** [comment]
@WandererFan, yes. The one with the drop down that allows selecting shortening or lengthening.

**Comment by WandererFan** [comment]
fixed as part of #14216.
